dataframe-jdbc/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/imdbTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.jetbrains.kotlinx.dataframe.io
+
+import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.annotations.DataSchema
+import org.jetbrains.kotlinx.dataframe.api.cast
+import org.jetbrains.kotlinx.dataframe.api.print
+import org.junit.Test
+import java.sql.DriverManager
+import java.util.*
+
+
+private const val URL = "jdbc:mariadb://localhost:3306/imdb"
```

Comment:
Yeah like you said, we probably need to run these only before a release, maybe on TC? In other cases we should probably disable the tests as it will throw unnecessary errors when building the library.